src/Adapter/PlatformServices.WinUI/Services/WinUIFileOperations.cs

```diff
@@ -108,7 +109,14 @@ public void DisposeNavigationSession(object navigationSession)
         /// </returns>
         public string GetFullFilePath(string assemblyFileName)
         {
-            return (SafeInvoke<string>(() => Path.Combine(Windows.ApplicationModel.Package.Current.InstalledLocation.Path, assemblyFileName)) as string) ?? assemblyFileName;
+            string packagedPath()
+            {
+                return SafeInvoke<string>(() => Path.Combine(Windows.ApplicationModel.Package.Current.InstalledLocation.Path, assemblyFileName)) as string ?? assemblyFileName;
+            }
+
+            return this.isPackaged
+                   ? packagedPath()
+                   : assemblyFileName;
```

Comment:
assemblyFileName == filename

but packaged path == path

odd to be mixing apples and oranges like that. That should be `Directory.Parent(assemblyName)` ?

Reply:
I'll double check this, but in unpackaged-apps and fullfw `assemblyFileName` is a full path. In UWP and WinUI it's a filename.